types/polka/tslint.json

```diff
@@ -1 +1,6 @@
-{ "extends": "dtslint/dt.json" }
+{
+  "extends": "dtslint/dt.json",
+  "rules": {
+    "no-outside-dependencies": false
```

Comment:
Is this needed?

Reply:
yes, otherwise `dtslint` fails with various errors because of referencing `node`s typings. The suggested fix that's linked to in the linter's error message doesn't work.

Reply:
Can you take a closer look at this, @marvinhagemeister? I don’t know what’s going on, but these typings _already_ referenced node before your changes without this rule, and every other package on DefinitelyTyped that references node doesn’t need this rule.